src/cco-modules/TimeOntology.ttl

```diff
@@ -49,7 +49,7 @@ cco:ont00001779 rdf:type owl:ObjectProperty ;
                  rdfs:domain obo:BFO_0000038 ;
                  rdfs:range obo:BFO_0000148 ;
                  rdfs:label "has inside instant"@en ;
-                 skos:definition "For Temporal Interval t1 and Temporal Instant t2, t1 has inside instant t2 if and only if there exists Temporal Instants t3 and t4 that are part of t1 and non-identical with t2, such that t3 is before t2 and t4 is after t2."@en ;
+                 skos:definition "x has_inside_instant y iff x is an instance of One Dimensional Temporal Region and y is an instance of Zero Dimensional Temporal Region and y is_inside_instant_of x."@en ;
```

Comment:
See comment below. I don't know that Allen defined relations involving instants.